vue/components/ui/atoms/svg-viewer/svg-viewer.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <object class="svg-viewer" type="image/svg+xml" v-bind:data="svgData" />
+</template>
+
+<style scoped></style>
+
+<script>
+export const SVGViewer = {
+    name: "svg-viewer",
+    props: {
+        src: {
+            type: String,
+            default: null
+        }
+    },
+    data: function() {
+        return {
+            svgData: null
+        };
+    },
+    watch: {
+        async src(value) {
+            await this.loadSVGData(value);
+        }
+    },
+    created: async function() {
+        await this.loadSVGData(this.src);
+    },
+    methods: {
+        async loadSVGData(src) {
+            if (!src) return;

```

Comment:
in case of `!src` it should still set `this.svgData` to null, otherwise you won't be able to clear out previous svgs